client/src/components/Common/Button/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import { Button } from '@material-ui/core';
+
+const ButtonComponent = props => {
+  const { className, value, onClick, type, disabled } = props;
+  return (
+    <Button
+      onClick={onClick}
+      type={type}
```

Comment:
remove `type` , `value` 
theses `props` that will be using for now 
```suggestion

      <Button
      onClick={onClick}
      variant={variant}
      className={className}
      disabled={disabled}
      fullWidth={fullWidth}
      color={color}
    >
      {children}
    </Button>
```

check 
https://material-ui.com/api/button/
https://material-ui.com/components/buttons/